converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/TableStorage/IExceptionStore.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+
+namespace Microsoft.Health.DicomCast.Core.Features.ExceptionStorage
```

Comment:
This should move to a folder `/Features/ExceptionStorage`. As a general rule the folder structure should match the namespace.

Reply:
Updated them all